src/main/java/org/openmrs/isanteplus/performancedata/options/GeneratorOptions.java

```diff
@@ -122,6 +130,9 @@ private void setOptions() {
                 case CommandConstants.VISITS_AMOUNT:
                     visitOption.setVisits(Long.parseLong(option.getValue()));
                     break;
+                case CommandConstants.ENCOUNTERS_AMOUNT:
+                    encounterOption.setEncounters(Long.parseLong(option.getValue()));
```

Comment:
is there a possibility that option.getValue() == null?

Reply:
No, the tool will stop an execution. I'll add a more precise message for the user.